extensions/ql-vscode/src/databaseFetcher.ts

```diff
@@ -445,6 +462,16 @@ export async function convertToDatabaseUrl(
   }
 }
 
+async function pullLgtmProject(lgtmUrl: string): Promise<any> {
```

Comment:
`pull` generally has a different meaning when dealing with git operations. This is really about getting the project's metadata. 

```suggestion
async function downloadLgtmProjectMetadata(lgtmUrl: string): Promise<any> {
```

or something like that.